features/providers/means_report.feature

```diff
@@ -1,385 +0,0 @@
-@javascript
```

Comment:
why are we removing this feature test or not replacing it with something? These were intended to exercise the conditional logic that controls differing display for the fundamental journey types. 

I was thinking to extend these tests as part of the non-means-tested work which requires more conditional display logic.

Reply:
Previously we relied on the `show_as_html` configuration in `wicked_pdf` to render the view in tests rather than generate a PDF file.

If we wanted to replicate the same behaviour now, we would need to wrap functionality in a conditional like this.

```ruby
def show
  @cfe_result = @legal_aid_application.cfe_result
  @manual_review_determiner = CCMS::ManualReviewDeterminer.new(@legal_aid_application)
  
  unless params.key?(:debug)
    html = render_to_string("show", layout: "pdf")
    pdf = Grover.new(html).to_pdf
    
    send_data pdf, filename: "means_report.pdf", type: "application/pdf", disposition: "inline"
  else
    render "show", layout: "pdf"
  end
end
```

It seemed roundabout, so I removed the tests completely.

#4725 explores how we might take a different approach to testing the rendered content of reports.

Reply:
If that will just work then it does sound more sensible given it unblocks ruby 3.1.3 and alpine 3.17 and we need to cover the non-means-tested changes in the coming sprint, so yes.